### PR TITLE
Added render pipeline error to build

### PR DESCRIPTION
**Pull request by SamCartledge**
**Sample**

Fixes User Experience when trying to build with both render pipelines installed

**Summary**

Added a pop-up dialog instructing the user to remove a render pipeline if it detects more than 1 render pipeline in the project after pressing Build. Also added logic to AutoImport.cs to switch the render pipeline asset if you removed a package the current render pipeline asset was referencing. Removed Debug logs from AutoImport

**Tests**

https://user-images.githubusercontent.com/42052635/176326491-e277ff75-bf51-434e-a71d-face25e6ab13.mp4
Tested on Windows and MacOS editor. Tested building for all platforms

**ArcGIS Maps SDK Version**

Latest daily version 3556

### CODE REVIEW

**Comment by Jade-JadeH** [comment]
I tested switching between HDRP and URP, packaging with both pipelines, and also removing one pipeline and readding it, all work well. 

When I switch HDRP and URP, there's a process running 
image
which I don't remember seeing before (I could be wrong), what does it do?

**Review by Jade-JadeH — APPROVED** [review]

File: samples_project/Assets/SampleViewer/Scripts/AutoImport.cs [hunk 1]
```diff
@@ -15,7 +16,19 @@ static CallReImport()
...
+            RenderPipelineAsset HDRPasset = AssetDatabase.LoadAssetAtPath<RenderPipelineAsset>("Assets/SampleViewer/Resources/SampleGraphicSettings/SampleHDRPipeline.asset");
+            GraphicsSettings.renderPipelineAsset = HDRPasset;
+
+#elif USE_URP_PACKAGE
+            RenderPipelineAsset URPasset = AssetDatabase.LoadAssetAtPath<RenderPipelineAsset>("Assets/SampleViewer/Resources/SampleGraphicSettings/SampleURPipeline.asset");
+            GraphicsSettings.renderPipelineAsset = URPasset;
+#elif USE_HDRP_PACKAGE
+            RenderPipelineAsset HDRPasset = AssetDatabase.LoadAssetAtPath<RenderPipelineAsset>("Assets/SampleViewer/Resources/SampleGraphicSettings/SampleHDRPipeline.asset");
+            GraphicsSettings.renderPipelineAsset = HDRPasset;
+#endif
```
Comment (mrnelson6) [inline samples_project/Assets/SampleViewer/Scripts/AutoImport.cs:29]:
```suggestion
#if USE_HDRP_PACKAGE
            RenderPipelineAsset HDRPasset = AssetDatabase.LoadAssetAtPath<RenderPipelineAsset>("Assets/SampleViewer/Resources/SampleGraphicSettings/SampleHDRPipeline.asset");
            GraphicsSettings.renderPipelineAsset = HDRPasset;
#elif USE_URP_PACKAGE
            RenderPipelineAsset URPasset = AssetDatabase.LoadAssetAtPath<RenderPipelineAsset>("Assets/SampleViewer/Resources/SampleGraphicSettings/SampleURPipeline.asset");
            GraphicsSettings.renderPipelineAsset = URPasset;
#endif
```

File: samples_project/Assets/Editor/NewAssemblyDef.asmdef [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (mrnelson6) [inline samples_project/Assets/Editor/NewAssemblyDef.asmdef:1]: If you follow my above suggestion we don't need to add this new assembly definition
> Can this file be put under the samples_project/Assets/SampleViewer/Scripts/ folder?

File: samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs [hunk 3]
```diff
@@ -43,7 +43,7 @@ private void Update()
         if (mapComponent != null && mapComponent.APIKey == "")
         {
             mapComponent.APIKey = APIKey;
-#if (UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX)
+#if (UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX || UNITY_IOS)
```
Comment (mrnelson6) [inline samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs:46]: This is still an issue? If so can you make an issue in SSV?

**Review by mrnelson6 — CHANGES_REQUESTED** [review]

File: samples_project/Assets/Editor/BuildSamplesError.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#if UNITY_EDITOR
```
[the remark this answers is not in the document]
Comment (SamCartledge) [reply samples_project/Assets/Editor/BuildSamplesError.cs:1]: I tried, and it was not being called at the correct time

File: samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs (hunk 3 above)
Comment (SamCartledge) [reply samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs:46]: Yes it is still an issue, I will make one in SSV

File: samples_project/Assets/Editor/NewAssemblyDef.asmdef (hunk 2 above)
Comment (mrnelson6) [reply samples_project/Assets/Editor/NewAssemblyDef.asmdef:1]: Ah ok, from your comment I see that the Editor folder is special. https://docs.unity3d.com/Manual/SpecialFolders.html

Can you rename this file to `EditorAssemblyDef.asmdef`

Comment (SamCartledge) [reply samples_project/Assets/Editor/NewAssemblyDef.asmdef:1]: Will do 👍

File: samples_project/Assets/Editor/EditorAssemblyDef.asmdef [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+	"name": "NewAssemblyDef",
```
Comment (mrnelson6) [inline samples_project/Assets/Editor/EditorAssemblyDef.asmdef:2]: This line needs to be updated to reflect the name of the file

File: samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs (hunk 3 above)
Comment (SamCartledge) [reply samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs:46]: made: https://devtopia.esri.com/runtime/ssv-normandy/issues/1519

File: samples_project/Assets/Editor/EditorAssemblyDef.asmdef (hunk 5 above)
Comment (SamCartledge) [reply samples_project/Assets/Editor/EditorAssemblyDef.asmdef]: hm, I had changed it but must not have saved, fixed

**Review by mrnelson6 — APPROVED** [review]
Looks good. Nice job @SamCartledge

**Pull request merged**